.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Ah and here is the PR for x-compilation https://github.com/fabricedesre/rustpi2/pull/2/files